### ISSUE TITLE
数据集划分

### ISSUE THREAD

**Issue by Yyang-sys** [G-pz/HPN-CR#2]
请问自己划分数据集，是按着以下格式吗？
+-- train
   +-- source_folder
     +-- s1
        |+--ROIs1970_fall_s1
          |+--s1_3
           --xxx.tif
           ......
         |+--s1_4
           --xxx.tif
           ......
         |.......
         |+--ROIs1158_spring_s1
                 |+--s1_3
                    --xxx.tif
                    ......
                 |+--s1_4
                    --xxx.tif
                    ......
        |.......

          |+--ROIs1868_summer_s1
                  |+--s1_3
                     --xxx.tif
                      ......
                  |+--s1_4
                     --xxx.tif
                      ......
                  |.......

           |+--ROIs2017_winter_s1
                  |+--s1_3
                     --xxx.tif
                      ......
                  |+--s1_4
                     --xxx.tif
                      ......
                  |.......

        +-- s2
          .......
        +-- s2_cloudy 
          ......

**Comment by G-pz** [comment]
There is no data.csv for the validation set and validation data is needed, you can generate one by following the same format as the data.csv for the test set.

**Comment by G-pz** [comment]
there is no data.csv for the validation set , you can generate one by following the same format as the data.csv for the test set.
